get_dependencies.py

```diff
@@ -24,16 +25,29 @@ def download(url, filename):
     sys.stdout.write('\n')
 
 
-print('[*] Extracting webdriver archives...')
-for f in ['geckodriver', 'nightly', 'chromedriver', 'chrome-linux']:
-    tar = tarfile.open('tools/%s.tar.xz' % f, 'r:xz')
-    tar.extractall(path='tools/')
-    tar.close()
+if sys.platform.startswith('linux'):
+    url = 'https://www.dropbox.com/s/cbg8mpg03vd8cph/linux.tar.xz?dl=1'
+    name = 'linux.tar.xz'
+elif sys.platform.startswith('darwin'):
+    url = 'https://www.dropbox.com/s/x5625gvbwaaxpnf/mac.tar.xz?dl=1'
+    name = 'mac.tar.xz'
+elif sys.platform.startswith('win32'):
+    url = 'https://www.dropbox.com/s/scmgetma3d8jtl6/win32.tar.xz?dl=1'
+    name = 'win32.tar.xz'
+
+print('[*] Downloading support files')
+download(url, name)
+print('[*] Extracting files')
+with tarfile.open(name) as f:
```

Comment:
Please either use the same code as before, or use the [TarFile class](https://docs.python.org/3/library/tarfile.html#tarfile-objects) that implements a context manager (so you don't have to close the file manually).

Reply:
tar = tarfile.open('tools/%s.tar.xz' % f, 'r:xz')
    tar.extractall(path='tools/')
    tar.close()
like this? as it is currently?

Reply:
Yes. Either this, or use the TarFile class, as you prefer.